### PR TITLE
DOCS-4552 Known issue in APID

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ These release notes cover the following versions of these features:
 == 2.4.6
 *April 20, 2019*
 
-This release includes two new features and several fixes.
+This release includes two new features and several fixes. There is also one known issue
```
Comment (mcherem) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Not sure what the general conventions are, but would suggest: This release includes two new features, several fixes, and one known issue.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 2]
```diff
@@ -43,6 +43,10 @@ This release includes two new features and several fixes.
 // SE-11175
 *  The API console might display all examples for all methods that were in a specification, not just the examples for the selected method.
```
Comment (mcherem) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Is 'could' here better than 'might'? I don't know the product, so just asking if the user has enough context here about the problem - e.g. all methods, all examples. I'd was also asking myself, why would this happen? Do we ever use 'You'?  Ex: "You could see all examples for all methods that were in a specification, not just the examples for a selected method in the API console". Now if this is the JIRA title for the customer and needs to stay as is, then please ignore my comments.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: I don't think that the items listed in your suggestion are parallel. Features and fixes are things that are items that are included in a release. They are planned, worked on, and tested. A known issue is not included in the same sense. It is instead something that is unplanned, not worked on, and not tested. A feature and a fix both seem to me to be additions, whereas a known issues seems to be a subtraction.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: That part has already been reviewed. The change in this PR is only the addition of the known issue.

Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Also, the current wording is more explicit and clear about the behavior of the editor.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 3]
```diff
@@ -43,6 +43,10 @@ This release includes two new features and several fixes.
...
 
+== Known Issue
+// DOCS-4552
+Because of a bug in a third-party library that is used for the validation of an XML against an XSD schema, the API editor can be inconsistent in displaying the error message `Could not validate XSD schema`. For example, you might or might not see the error message whenever you open a particular file that contains XML.
```
Comment (mcherem) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: For the part, "...an XML", I feel like it reads like XML should be a qualifier. Like an XML what? Or can 'an' be removed? But again, I don't know the product.
The last sentence is a tough one - how do you tell users that they can 'sometimes' see a message or 'sometimes not'. Can't really think of a much better way to reword at this moment.

Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: "An" was left over from the initial description that the dev sent me.

**Pull request merged**